dotnetv3/BedrockRuntime/Actions/HelloBedrockRuntime.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved. 
+// SPDX-License-Identifier:  Apache-2.0
+
+namespace BedrockRuntimeActions
+{
+    internal class HelloBedrockRuntime
```

Comment:
I see this hello example, should it also be added to the metadata under `bedrock-runtime_Hello` ?

Reply:
I've created this so that someone can run it locally. I haven't added snippet as the Java version is not added neither. I'm glad to revisit if you think it brings value.